src/simulation/dynamics/gravityEffector/_UnitTest/test_gravityDynEffector.py

```diff
@@ -511,6 +512,7 @@ def multiBodyGravity(show_plots):
     allGrav2.gravBodies = gravityEffector.GravBodyVector([gravBody1, gravBody2])
     allGrav2.linkInStates(newManager)
     allGrav2.registerProperties(newManager)
+    allGrav2.Reset(0)
```

Comment:
Why are you calling Reset() here?

Reply:
Now, `Reset` does some setting up, calling `initBody`, which in turn initializes dependent parameters for each body. We have to call it manually here (similar to `linkInStates` and `registerProperties`) since we are using the module in a special testing context. Usually, the simulation will call `Reset`.